instrumentation/ratpack-1.4/javaagent/ratpack-1.4-javaagent.gradle

```diff
@@ -20,3 +20,7 @@ dependencies {
     testImplementation group: 'com.sun.activation', name: 'jakarta.activation', version: '1.2.2'
   }
 }
+
+tasks.withType(Test) {
+  jvmArgs "-Dio.opentelemetry.javaagent.shaded.io.opentelemetry.context.enableStrictContext=false"
```

Comment:
does this mean that our ratpack instrumentation actually leaks context, or is it an artifact of something else? If we are really leaking contexts, should we create issues to fix all of these, so we make sure to circle back and fix them?

Reply:
Good point, filed an issue